extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.css

```diff
@@ -277,7 +277,7 @@ amp-lightbox-gallery .amp-carousel-button {
 .i-amphtml-lightbox-gallery-trans {
   pointer-events: none !important;
   position: fixed !important;
-  z-index: 1001 !important;
+  z-index: 2147483642 !important;
   top: 0 !important;
```

Comment:
Since with the animation library, we no longer need this element to be sized to the viewport size anymore, let's see if we can move the `top`, `bottom`, `left`, `right` CSS for this and reduce some bloat?

Reply:
Reworked this a bit and removed the transition layer element completely.